src/components/calculator/controls/SegmentedControl.tsx

```diff
@@ -54,13 +57,15 @@ export const SegmentedControl: React.FunctionComponent<{
   const hoverDescIfPresentOtherwiseActiveDesc =
     hoverDesc !== ''
       ? hoverDesc
-      : (activeValue !== '' && props.source[activeValue].value) || ''
+      : props.useActiveDesc && activeValue !== ''
```

Comment:
I find nested ternaries a little hard to read, but I'm not fussed enough to insist it changes.